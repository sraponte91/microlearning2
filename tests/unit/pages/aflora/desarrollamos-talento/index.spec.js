import { config, createLocalVue, mount } from '@vue/test-utils'
import VueMeta from 'vue-meta'
import AfloraDesarrollamosTalento from '@/pages/aflora/desarrollamos-talento/index.vue'
// create vue with vue-meta
const localVue = createLocalVue()
localVue.use(VueMeta, { keyName: 'head' })

const mock = {
    localVue,
    mocks: {}
}

const wrapper = mount(AfloraDesarrollamosTalento, mock)

describe('index.vue', () => {
    test('Is a Vue instance', () => {
        expect(wrapper.vm).toBeTruthy()
    })
    test('Renders correctly', () => {
        expect(wrapper.element).toMatchSnapshot();
    });
    test('has correct <head> content', () => {
        expect(wrapper.vm.$metaInfo.title)
            .toBe(config.mocks['$afloraData'].afloraDesarrollamosTalento.meta.title +
                ' - ' + config.mocks['$constants'].home.meta.title)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'title'
        ).content)
            .toBe(config.mocks['$afloraData'].afloraDesarrollamosTalento.meta.metaTitle +
                ' - ' + config.mocks['$constants'].home.meta.title)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'description'
        ).content).toBe(config.mocks['$afloraData'].afloraDesarrollamosTalento.meta.metaDescription)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'og:title'
        ).content)
            .toBe(config.mocks['$afloraData'].afloraDesarrollamosTalento.meta.ogTitle +
                ' - ' + config.mocks['$constants'].home.meta.title)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'og:description'
        ).content).toBe(config.mocks['$afloraData'].afloraDesarrollamosTalento.meta.ogDescription)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'og:image'
        ).content).toBe(config.mocks['$constants'].baseURL + config.mocks['$afloraData'].afloraDesarrollamosTalento.meta.ogImage)
    })
})
