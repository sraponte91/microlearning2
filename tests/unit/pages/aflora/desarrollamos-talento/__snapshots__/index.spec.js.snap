// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.vue Renders correctly 1`] = `
<main>
  <div
    class="breadc-bak"
  >
    <div
      class="container margin-0-auto"
    >
      <ol
        class="breadcrumb ff-sans-r mb-2"
      >
        <li
          class="breadcrumb-item"
        >
          <a
            class=""
            href="/"
            target="_self"
          >
            Home
          </a>
        </li>
         
        <li
          class="breadcrumb-item"
        >
          <a
            class=""
            href="/aflora"
            target="_self"
          >
            Aflora
          </a>
        </li>
         
        <li
          class="breadcrumb-item active"
        >
          <span
            aria-current="location"
          >
            Desarrollamos talento
          </span>
        </li>
      </ol>
       
      <div
        class="back-to-home position-relative d-flex align-items-center small-size"
      >
        <i
          class="ic-back"
        />
         
        <a
          class="ff-sans-b"
          href="javascript:void(0)"
        >
          Volver
        </a>
      </div>
    </div>
  </div>
   
  <section
    class="ig-tx-zing-zag desarrolla bg-white"
  >
    <div
      class="container"
    >
      <div
        class="sm-siz-container margin-0-auto"
      >
        <div
          class="d-flex multiple-reapt align-items-start flex-wrap-991"
        >
          <h2
            class="ff-sans-b ss d-block-991 d-none-n wow bounceInLeft"
            data-wow-duration="2s"
          >
            
            Desarrollamos 
            <br />
            
            talento
          
          </h2>
           
          <div
            class="txt-points position-relative order-2-991 wow bounceInLeft"
            data-wow-duration="2s"
          >
            <h2
              class="ff-sans-b d-none-991"
            >
              
              Desarrollamos 
              <br />
              
              talento
            
            </h2>
             
            <div
              class="pra-p for-bgcol"
            >
              <p>
                
                En Aflora brindamos a todas las organizaciones sociales con
                gran potencial una ruta de formación para fortalecer sus
                capacidades y las de sus equipos. Buscamos ayudarlas a
                identificar sus oportunidades de mejora en distintas áreas a
                través de un autodiagnóstico. Nuestro propósito es ayudarlas a
                identificar los medios para lograr su sostenibilidad
                financiera, ambiental y social.
              
              </p>
               
              <p>
                
                Trabajamos unidos con las organizaciones, para impulsar sus
                causas, cerrar brechas sociales y contribuir a la construcción
                de un mejor país.
              
              </p>
            </div>
          </div>
           
          <div
            class="img-points order-1-991 wow bounceInRight"
            data-wow-duration="2s"
          >
            <img
              alt=""
              class="object-fit-cover"
              src="/images/programa-aflora/3-desarrollamos-talento/desarrollamostalento_bloque1.jpg"
            />
          </div>
        </div>
         
        <div
          class="d-flex multiple-reapt align-items-start pp-lr flex-wrap-767"
        >
          <div
            class="oly-img-bx wow bounceInLeft"
            data-wow-duration="2s"
          >
            <img
              alt=""
              class="object-fit-cover"
              src="/images/programa-aflora/3-desarrollamos-talento/desarrollamostalento_bloque2_destacado.jpg"
            />
          </div>
           
          <div
            class="oly-tx-bx wow bounceInRight"
            data-wow-duration="2s"
          >
            <div
              class="inn-bx-bg ff-sans-b"
            >
              <p>
                
                Conozca los ciclos de formación, y certifíquese de manera
                gratuita con prestigiosas universidades.
              
              </p>
               
              <div
                class="links-to-r text-right"
              >
                <a
                  to="/aflora/desarrollamos-talento/ciclos-de-formacion"
                >
                  Quiero conocer más &gt;
                </a>
              </div>
            </div>
          </div>
        </div>
         
        <div
          class="levels-nd-benef text-center"
        >
          <h2
            class="ff-sans-b"
          >
            Niveles y beneficios por nivel
          </h2>
           
          <div
            class="pra-p"
          >
            <p>
              
              Ofrecemos un conjunto de capacitaciones y ciclos de formación
              adaptados a las condiciones, necesidades y características de
              cada organización. Luego de diligenciar el autodiagnóstico, las
              organizaciones serán clasificadas en uno de los 4 niveles que
              tiene Aflora. 
              <br />
              

              Cada nivel está construido de manera estratégica para brindarle
              a las organizaciones los contenidos que necesitan según el
              estado real en el que se encuentra cada área de la organización.
              
              <br />
              

              El objetivo es que las organizaciones cumplan con los requisitos
              de cada nivel, para que logren avanzar al siguiente y así
              terminar con éxito todo el proceso de aprendizaje y mejora.
              
              <br />
              

              Estas son las características de cada nivel: 
              <br />
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>
   
  <section
    class="school-directors"
  >
    <div
      class="container"
    >
      <div
        class="sm-siz-container margin-0-auto"
      >
        <div
          class="position-relative for-mt-mins"
        >
          <div
            class="im-prt"
          >
            <img
              alt=""
              class="object-fit-cover"
              src="/images/programa-aflora/3-desarrollamos-talento/escuela_de_directores_desktop.jpg"
            />
          </div>
           
          <div
            class="txt-sec margin-0-auto"
          >
            <div
              class="text-center"
            >
              <h3
                class="ff-sans-b text-white"
              >
                Escuela de directores
              </h3>
            </div>
             
            <div
              class="pra-p text-white"
            >
              <p>
                
                Basados en el método del Caso, desarrollamos 8 modelos a
                partir de experiencias organizacionales de algunos de los
                participantes en el programa para dar una visión particular y
                de conjunto de las organizaciones, así como desarrollar
                pensamiento estratégico y algunas de las habilidades,
                capacidades y competencias directivas necesarias para el
                desempeño profesional frente a una fundación.
              
              </p>
               
              <p>
                
                Si deseas ampliar la información de la Escuela de Directores,
                conocer el pénsum, el cronograma y los costos, entre otros, te
                invitamos a visitar el sitio web haciendo clic en el siguiente
                botón
              
              </p>
            </div>
             
            <div
              class="btn-g text-center"
            >
              <a
                class="btn-know-leave ff-sans-b"
                href="/escuela-de-directores-landing/"
              >
                Conocer Niveles
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</main>
`;
